biome.jsonc

```
@@ -9,46 +9,51 @@
   "vcs": {
     "defaultBranch": "unstable"
   },
-  "files": {
-    "include": ["packages/*/src/**/*.ts", "packages/*/test/**/*.ts", "configs/**/*.ts", "./vitest.config.ts"]
-  },
-  "formatter": {
-    "ignore": ["**/lib", "**/.nyc_output", "./packages/*/spec-tests", "**/node_modules", "./packages/*/node_modules/**"]
-  },
-  "javascript": {
-    "globals": ["Bun"]
+  "assist": {
+    "actions": {
+      "source": {
+        "organizeImports": { "level": "off", "options": { "identifierOrder": "lexicographic" } },
+        "useSortedKeys": { "level": "off", "options": { "sortOrder": "lexicographic" } }
+      }
+    }
   },
-  "organizeImports": {
-    "enabled": true
+  "files": {
+    "includes": [
+      "**/packages/**/*/src/**/*.ts",
+      "**/packages/**/*/test/**/*.ts",
+      "**/configs/**/*.ts",
+      "vitest.config.ts",
+      "!**/lib",
+      "!packages/**/*/spec-tests",
+      "!packages/**/*/node_modules/"
+    ]
   },
   "linter": {
     "rules": {
       "correctness": {
         "useImportExtensions": {
           "level": "error",
           "options": {
-            "suggestedExtensions": {
-              "ts": {
-                "module": "js",
-                "component": "jsx"
-              }
-            }
+            "forceJsExtensions": false
           }
+        },
+        "useParseIntRadix": {
+          "level": "off"
         }
       },
       "performance": {
         // This rule should be enabled but with considerations and careful review
         "noDelete": "off"
       },
       "style": {
-        // The code usage looks suspicious so it should be enabled in a separate PR
-        "noCommaOperator": "off",
+        // Will be enabled in a separate PR
+        "useArrayLiterals": "off",
         // There are a lot of places we mutate params, should be fixed in an independent PR.
         "noParameterAssign": "off",
         "noRestrictedGlobals": {
           "level": "error",
           "options": {
-            "deniedGlobals": ["fetch"]
+            "deniedGlobals": { "fetch": "TODO: Add a custom message here." }
```

before we switched to biome we had this message

```suggestion
            "deniedGlobals": { "fetch": "Please use 'fetch' from '@lodestar/api' instead." }
```